src/registries/npm-registry.js

```diff
@@ -218,21 +221,29 @@ export default class NpmRegistry extends Registry {
     }
   }
 
-  getScope(packageName: string): string {
-    return !packageName || packageName[0] !== '@' ? '' : packageName.split(/\/|%2f/)[0];
+  getScope(packageIdent: string): string {
+    // removing escaped / from package names in full paths
+    packageIdent = packageIdent.replace('%2f', '/');
```

Comment:
I'm trying to say this normalization should not be done here, and actually should not be done at all. `http://foo.bar:80/@scope%2fbar/baz` is not equivalent to `http://foo.bar:80/@scope/bar/baz`. See https://stackoverflow.com/a/14658209/90297